src/scenes/start.js

```diff
@@ -18,31 +18,53 @@ export default class StartScene extends Phaser.Scene {
     }
 
     create() {
-        this.start_player = this.physics.add.sprite(100, options.windowHeight - 100, "codey");
+        this.createTextElements();
```

Comment:
Moving `this.createTextElements` to the beginning of `create()` meant Codey gets drawn on top of them.